Source/SubtitleParser.cpp

```diff
@@ -0,0 +1,71 @@
+#include "pch.h"
+#include "SubtitleParser.h"
+#include "SubtitleParser.g.cpp"
+#include "FFmpegMediaSource.h"
+#include "MediaSourceConfig.h"
+
+namespace winrt::FFmpegInteropX::implementation
+{
+    IAsyncOperation<winrt::FFmpegInteropX::SubtitleParser> SubtitleParser::ReadSubtitleAsync(winrt::Windows::Storage::Streams::IRandomAccessStream stream,
+        hstring streamName,
+        winrt::FFmpegInteropX::MediaSourceConfig config,
+        winrt::Windows::Media::Core::VideoStreamDescriptor videoDescriptor)
+    {
+        winrt::apartment_context caller; // Capture calling context.
+        co_await winrt::resume_background();
+
+        auto parser = co_await winrt::FFmpegInteropX::implementation::FFmpegMediaSource::ReadExternalSubtitleStreamAsync(stream, streamName, config, nullptr, nullptr, 0, false);
```

Comment:
I think the videoDescriptor must be passed in here.

Reply:
Fixed